bmv2/Makefile.am

```diff
@@ -147,6 +150,49 @@ if P4THRIFT
     AM_CPPFLAGS += -DP4THRIFT
 endif
 
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+unused_for_now = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+of_sources = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c
+
+of_includes = \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+plugin_files += $(of_sources) $(of_includes)
+plugin_headers += $(of_includes)
+
+pdlib_LTLIBRARIES += libof.la
+
+libof_la_SOURCES = $(of_sources) $(of_includes)
+
+# Flags for including from p4ofagent/indigo
+AM_CPPFLAGS += -D_BMV2_
+AM_CPPFLAGS += -I $(builddir)/../../../../../../submodules/p4ofagent
```

Comment:
I don't know if you are submitting this Pull Request for review but just in case:
The p4-build repo is standalone and as a result it is not acceptable to assume it is used within switch, within p4factory. Required OF headers should either be installed (and `configure` should check for existence) or the path should be provided to `configure`.